moveit_ros/occupancy_map_monitor/CMakeLists.txt

```diff
@@ -15,54 +23,59 @@ if(APPLE)
   find_package(X11 REQUIRED)
 endif(APPLE)
 
-find_package(catkin REQUIRED COMPONENTS
-  moveit_core
-  moveit_msgs
-  pluginlib
-)
-
+find_package(moveit_core REQUIRED)
+find_package(moveit_msgs REQUIRED)
+find_package(pluginlib REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(octomap REQUIRED)
-
-catkin_package(
-  INCLUDE_DIRS
-    include
-  LIBRARIES
-    ${MOVEIT_LIB_NAME}
-  CATKIN_DEPENDS
-    moveit_core
-    moveit_msgs
-  DEPENDS
-    EIGEN3
-    OCTOMAP
-)
+find_package(geometric_shapes REQUIRED)
 
 include_directories(include
                     ${Boost_INCLUDE_DIRS}
-                    ${catkin_INCLUDE_DIRS}
+                    ${rclcpp_INCLUDE_DIRS}
+                    ${moveit_core_INCLUDE_DIRS}
+                    ${moveit_msgs_INCLUDE_DIRS}
+                    ${geometric_shapes_INCLUDE_DIRS}
                     )
 include_directories(SYSTEM
                     ${EIGEN3_INCLUDE_DIRS}
                     ${X11_INCLUDE_DIR}
                     )
 
-add_library(${MOVEIT_LIB_NAME}
+add_library(${MOVEIT_LIB_NAME} SHARED
   src/occupancy_map_monitor.cpp
   src/occupancy_map_updater.cpp
   )
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+ament_target_dependencies(${MOVEIT_LIB_NAME} rclcpp moveit_core moveit_msgs pluginlib octomap geometric_shapes)
+target_link_libraries(${MOVEIT_LIB_NAME} ${Boost_LIBRARIES})
 
 add_executable(moveit_ros_occupancy_map_server src/occupancy_map_server.cpp)
-target_link_libraries(moveit_ros_occupancy_map_server ${MOVEIT_LIB_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+ament_target_dependencies(moveit_ros_occupancy_map_server rclcpp moveit_core moveit_msgs pluginlib octomap geometric_shapes)
+target_link_libraries(moveit_ros_occupancy_map_server ${MOVEIT_LIB_NAME} ${Boost_LIBRARIES})
```

Comment:
```suggestion
target_link_libraries(moveit_ros_occupancy_map_server ${MOVEIT_LIB_NAME})
```
Same

Reply:
Done adding `Boost` to `ament_target_dependencies`.